Documentation/project-docs/contributing.md

```diff
@@ -131,6 +109,28 @@ The following file header is the used for .NET Core. Please use it for new files
 - See [class.cpp](../../src/vm/class.cpp) for an example of the header in a C++ file.
 - See [List.cs](../../src/mscorlib/src/System/Collections/Generic/List.cs) for an example of the header in a C# file.
 
+Contributing Ports
+------------------
+
+We encourage ports of CoreCLR to other platforms. There are multiple ports ongoing at any one time. You may be interested in one of the following ports:
+
+Chips:
+
+- [ARM32](https://github.com/dotnet/coreclr/labels/arch-arm32)
+- [ARM64](https://github.com/dotnet/coreclr/labels/arch-arm64)
+- [X86](https://github.com/dotnet/coreclr/labels/arch-x86)
+
+Operating System:
+
+- [Linux](https://github.com/dotnet/coreclr/labels/os-linux)
```

Comment:
It could be that I could use better wording ... My point was "if you are interested in working on focusing on one of the ports, here they are." Someone might be specifically interested in Linux. My goal was to help to get into that space quickly. I realize that the port is "done". Initially, I just listed chips. That seemed lackluster.